bin/circleci/build-frontend.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -ex
 NODE_ENV=production ENABLE_PONTOON=1 ENABLE_DEV_CONTENT=1 ENABLE_DEV_LOCALES=1 npm run static
-zip -r $CIRCLE_ARTIFACTS/frontend.zip frontend/build
+zip -r $CIRCLE_ARTIFACTS/frontend.zip frontend/build dist
```

Comment:
Just a tweak here to include the production-style build in the artifact, after gulp-rev-all hashes filenames.